rs/nns/handlers/lifeline/unreleased_changelog.md

```diff
@@ -10,6 +10,8 @@ on the process that this file is part of, see
 ## Added
 
 ## Changed
+* Compile the `lifeline` canister with the latest Motoko compiler (0.16.3) (compared to compiling with 0.8.7 before)
+  but keep using classical orthogonal persistence by compiling with `--legacy-persistence`.
```

Comment:
I guess technically, this is not a behavior change, but it's a significant change in the "foundation", and so, deserves an entry like this. Maybe, call out the fact that there is no behavior change; just a foundation change.